### PR TITLE
SOL-14426 Updated the Version for Crypto.

### PR DESCRIPTION
**Pull request by Eknath-Sable**
Updated the Version for Crypto to v0.17.0

### CODE REVIEW

**Review by ratneshdubeysolidatus — APPROVED** [review]
LGTM

**Review by mix4242 — CHANGES_REQUESTED** [review]
Please remember to bump monstache version
Also we seem to be keeping a CHANGELOG in the `Solidatus-README.md` file

**Comment by Eknath-Sable** [comment]
> Please remember to bump monstache version Also we seem to be keeping a CHANGELOG in the `Solidatus-README.md` file

Added comment and bumped monstache version to 6.7.10+5

File: Solidatus-README.md [hunk 1]
```diff
@@ -28,3 +28,7 @@ Golang has 1 vulnerability CVE-2023-39320 which was fixed in 1.21.3 hence manual
 # 2024-01-10
 
 Golang has 2 vulnerability CVE-2023-39326 and CVE-2023-45285 which was fixed in 1.21.6 hence manually updated go.mod to use 1.21.6. (latest on main branch was 1.21.3)
+
+# 2024-05-07
+
+Golang has 1 vulnerability CVE-2023-48795 which was fixed in golang.org/x/crypto v0.17.0 hence manually updated go.mod to use 1.21.6. (latest on main branch was 1.21.3)
```
Comment (mix4242) [inline Solidatus-README.md]: The second half of this sentence doesn't sound accurate 👀 

May be worth also explaining why `golang.org/x/sys` & `golang.org/x/text` were bumped (I'm assuming as they had to be when `crypto` was bumped?)

**Review by mix4242 — CHANGES_REQUESTED** [review]

**Review by mix4242 — APPROVED** [review]

**Review by ratneshdubeysolidatus — APPROVED** [review]

**Pull request merged**